index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCUTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="movies, theater, film, theatre">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" type="image/png" href="http://example.com/myicon.png">
+    <title>Theater App</title>
+</head>
+
+
+<body>
+    <!--Make a header called Theater App name and a nav bar with different options user can click-->
+    <div class="main-wrapper">
+        <header>
+            <h1>Theater App</h1>
+            <nav class="navbar">
+                <ul>
+                    <!--Make 'browse movies' a drop down-->
+                    <li class="navbarlist"><a href="browse.html"></a>Browse Movies</li>
+                    <!--Make 'get tickets' a drop down-->
+                    <li class="navbarlist">Get tickets</li>
+                        <li class="navbar_dropdown_content">
```

Comment:
choose a consistent css class naming scheme. Some classes are like navbarlist, some are like navbar_dropdown_content, some are like img-slider. Choose between using - or _ to separate words and stick to it throughout the project